### PR TITLE
Add person as rightholder through backend endpoint

### PR DESCRIPTION
**Pull request by allinox**
image

## Description

This PR connects the GUI with the backend endpoint for adding people, thus new people can now be added, just like with organizations.

To this end, the addConnection endpoint in the BFF has been expanded, to include people input. Users only has a fixed number of times they can attempt to add a person with a valid SSN but mismatching surname before they are shut out from further attempts. 

## Related Issue(s)
- https://github.com/Altinn/altinn-authorization-tmp/issues/1331

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

## Summary by CodeRabbit

* **New Features**
  * Add individuals as right holders using person identifier + last name; endpoint now returns the created right-holder UUID.

* **Improvements**
  * Stronger client‑ and server‑side validation, clearer error responses (including 429 rate‑limit handling).
  * Updated UI: spinner and loading text while adding right holders.
  * Added localized validation messages and enhanced API documentation annotations.

* **Tests**
  * Expanded test coverage for organization‑ and person‑based add flows and validation edge cases.

### CODE REVIEW

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Integration/Clients/ConnectionClient.cs [hunk 1]
```diff
@@ -50,20 +52,25 @@ public ConnectionClient(
...
 
             if (!httpResponse.IsSuccessStatusCode)
             {
                 _logger.LogError($"Unexpected http response. Status code: {httpResponse.StatusCode}, Reason: {httpResponse.ReasonPhrase}");
                 throw new HttpStatusException("Unexpected http response.", "Unexpected http response.", httpResponse.StatusCode, null, httpResponse.ReasonPhrase);
             }
 
-            return httpResponse;
+            AddConnectionResponse response = JsonSerializer.Deserialize<AddConnectionResponse>(content, new JsonSerializerOptions { PropertyNameCaseInsensitive = true });
+            return response.Id;
```
Comment (sonwit) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Integration/Clients/ConnectionClient.cs]: Trying to add a new person, i'm passed to a new page, but i don't get a valid connection.

It looks like "ConnectionId" rather than the "PersonId" that get s returned? Should this instead be:  

```suggestion
            return response.ToId;
```

Comment (allinox) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Integration/Clients/ConnectionClient.cs]: Oops! 🙈
I thought the reason why I didn't get a valid page was because the connectionAPI was unstable, but you're right - it absolutely should be the `ToId` and not the `Id` that should be used here!
Thank you for noticing! 🙌

**Review by sonwit — APPROVED** [review]
Nice. 
Thanks for fixing the validation-messages! 
And the redirect works like a charm now! 
👍🏼
